fecgov/fec-cms
# Update Vice chair title on homepage, make it so this is auto-updates in the future.

## Summary 
- This adds an argument to the `commissioners.py` templatetag  so it can be used in multiple locations including the home page commissioners' listing. 
- With this PR, the homepage commissioner listing will be  updated automatically when their Wagtail commissioner page is edited (as on [leadership-and-structure](leadership-and-structure) and [leadership-and-structure/commissioners](https://www.fec.gov/about/leadership-and-structure/commissioners/) ). We were currently  hardcoding html for home page commissioners' display. 
- Added comments to explain how to use the argument to control the grid layout of the rendered templatetag ( i.e : 2 grid or 3 grid layout )

- Resolves #3308

## Impacted areas of the application
This impacts the commissioners block that appears in streamfields (currently used on about/leadership-and-structure ). Also effects the homepage commissioners' listing.
 - modified:   home/templates/home/home_page.html
 - modified:   home/templates/partials/current-commissioners.html
 - modified:   home/templatetags/home_page.py
 - modified:   home/templates/blocks/commissioners.html


Looks exactly the same as on production, but its not hardcoded:
<img width="1279" alt="Screen Shot 2019-10-30 at 2 13 08 AM" src="https://user-images.githubusercontent.com/5572856/67833419-f30bab00-faba-11e9-9998-11dc61fabab5.png">


- _Include a screenshot of the new/updated features in context (“in the wild”). If it is an interface or front end change, include both a Before and After screenshot._

## Related PRs
- The current  PR takes work done in this PR (https://github.com/fecgov/fec-cms/pull/3131) one step further.
https://github.com/fecgov/fec-cms/pull/3131/files#
- The above related-PR was an enhancement to the logic for this PR (https://github.com/fecgov/fec-cms/pull/963 )to capture term expiration dates and remove commissioners no longer active in addition to vacant seats(I think)


## How to test
- Checkout and run branch
- Verify that the homepage commissioner listing for Vice Chair Hunter includes her title and that others  are correct. (Your local Wagtail page for Hunter must have her title added if it is not already)
- You may not see the images if you do not have them on your local CMS, you can add the these three images to fec-cms/fec/media/images
![headshot--weintraub original](https://user-images.githubusercontent.com/5572856/67833899-52b68600-fabc-11e9-8ac9-fcb2f6d7fb20.png)
![headshot--walther original](https://user-images.githubusercontent.com/5572856/67833900-52b68600-fabc-11e9-9a20-76c6ef8dcb14.png)
![headshot--hunter original](https://user-images.githubusercontent.com/5572856/67833901-534f1c80-fabc-11e9-9dfd-6eed4f9bde88.png)



____
